### PR TITLE
feat: Updates to the file page TAROT-2708

### PR DESCRIPTION
**Pull request by nicklem**
✅ Ready to merge - awaiting feature launch
---

Updates the Files page documentation to match the migrated UI.

### :eyes: Live preview
https://tarot-2708-doc-updates-to-the-file--docs-codacy.netlify.app/repositories/files/

### :construction: To do
- [x] Screenshots
- [x] Fix all in-code TODOs
- [x] Self-review to match page in dev
- [x] Peer review

### CODE REVIEW

**Review by mrfyda — APPROVED** [review]

File: docs/repositories/files.md [hunk 1]
```diff
@@ -25,34 +25,23 @@ Use the search box to filter the list and find specific files:
...
-Click a specific file to see more detailed analysis information for that file, including statistics on:
+Click a specific file to see more detailed analysis information for that file.
 
--   **Size:** Lines of code, source lines of code, and commented lines of code
--   **Structure:** Number of methods and ratio of source lines of code per number of methods
--   **Complexity:** Complexity and complexity per method
--   **Duplication:** Number of clones and duplicated lines of code
-
-The button **Ignore File** allows you to ignore the selected file on future Codacy analysis.
+The header of the file detail page displays the same code quality metrics as the Files page, along with an **Ignore file** link to ignore the file, and a link to view the file on your Git provider.
```
Comment (claudiacarpinteiro) [inline docs/repositories/files.md:30]: We could keep the consistency with [other pages describing similar links](https://github.com/codacy/docs/blob/master/docs/repositories/pull-requests.md?plain=1#L22-L26) in the header. 

```suggestion
The header of the file detail page displays the same code quality metrics as the Files page, along with an **Ignore file** link to ignore the file, and a link to view the file on your Git provider.
```

File: docs/repositories/files.md [hunk 2]
```diff
@@ -25,34 +25,23 @@ Use the search box to filter the list and find specific files:
...
 
 ![File detail](images/files-details.png)
 
 Depending on the available analysis information for the file, Codacy displays one or more of the following tabs:
```
Comment (claudiacarpinteiro) [inline docs/repositories/files.md:34]: Are we dropping **Complexity/Method** metric? 🤔 
I noticed we don't have this value on the header, and we also don't have a Complexity tab...

**Review by claudiacarpinteiro — APPROVED** [review]
I left of couple of comments, but already looks good to go! 🚀

File: docs/repositories/files.md (hunk 2 above)
Comment (mrfyda) [reply docs/repositories/files.md:34]: I believe all of these are being dropped:

- Lines of code
- Source lines of code
- Commented lines of code
- Number of Methods
- sLoC / Method
- Complexity / Method
- Duplicated lines of code

**Pull request merged**